asmcli/lib/util.sh

```diff
@@ -141,6 +141,16 @@ prompt_user_for_value() {
   fi
 }
 
+prompt() {
```

Comment:
What do you think about "prompt_default_no" just to make it a bit clearer?

Reply:
Done